### PR TITLE
RN(7.10.4): write bug fix list for 7.10.4 release

### PR DESCRIPTION
**Pull request by educhastenier**
and migration tool release notes

### CODE REVIEW

**Comment by danila-m** [comment]
The release note for the migration tool needs to be done on 7.7 branch, not 7.10. Will edit this PR, and open a new one.

**Comment by educhastenier** [comment]
> The release note for the migration tool needs to be done on 7.7 branch, not 7.10. Will edit this PR, and open a new one.

Ok, nice catch. This is a bit weird, that's why I fooled myself

File: md/release-notes.md [hunk 1]
```diff
@@ -132,6 +132,13 @@ In versions 7.9.1-7.9.x the provided themes were embedding font-awesome. The lib
 * BST-171 Can't share a project on github using HTTPS and double authentication factor
 * BST-341 On MacOS, Studio installer might not detect the installed JRE
 
+### Fixes in Bonita 7.10.4 (2020-04-02)
+#### Fixes in Bonita Development Suite (Studio and UI Designer)
+* UID-253 - we should not be able to switch a widget to tab widget
+* UID-244 - Mac os .DS_Store prevents UID to start
+* UID-243 - UI designer fails to starts when page workspace does not contain folders only
```
Comment (NathalieC) [inline md/release-notes.md]:
```suggestion
* UID-243 - UI Designer fails to start when pages workspace does not contain folders only
```

File: md/release-notes.md [hunk 2]
```diff
@@ -132,6 +132,13 @@ In versions 7.9.1-7.9.x the provided themes were embedding font-awesome. The lib
 * BST-171 Can't share a project on github using HTTPS and double authentication factor
 * BST-341 On MacOS, Studio installer might not detect the installed JRE
 
+### Fixes in Bonita 7.10.4 (2020-04-02)
+#### Fixes in Bonita Development Suite (Studio and UI Designer)
+* UID-253 - we should not be able to switch a widget to tab widget
```
Comment (NathalieC) [inline md/release-notes.md]:
```suggestion
* UID-253 - Switching a widget to a tab widget should not be possible
```

File: md/release-notes.md [hunk 3]
```diff
@@ -132,6 +132,13 @@ In versions 7.9.1-7.9.x the provided themes were embedding font-awesome. The lib
...
+* UID-244 - Mac os .DS_Store prevents UID to start
+* UID-243 - UI designer fails to starts when page workspace does not contain folders only
+* STUDIO-3477 - Deploy fails when launched after a git update containing a BDM change used in updated process
+
```
Comment (NathalieC) [inline md/release-notes.md:141]: Bugs should be ordered by ASC jira order.
Studio bugs before UID bugs.

File: md/release-notes.md [hunk 4]
```diff
@@ -132,6 +132,13 @@ In versions 7.9.1-7.9.x the provided themes were embedding font-awesome. The lib
 * BST-171 Can't share a project on github using HTTPS and double authentication factor
 * BST-341 On MacOS, Studio installer might not detect the installed JRE
 
+### Fixes in Bonita 7.10.4 (2020-04-02)
+#### Fixes in Bonita Development Suite (Studio and UI Designer)
+* UID-253 - we should not be able to switch a widget to tab widget
+* UID-244 - Mac os .DS_Store prevents UID to start
+* UID-243 - UI designer fails to starts when page workspace does not contain folders only
+* STUDIO-3477 - Deploy fails when launched after a git update containing a BDM change used in updated process
```
Comment (NathalieC) [inline md/release-notes.md:140]: The jira bug resolution is "unresolved" but its status is "resolved". This is no easy query to perform. Good catch.

**Review by NathalieC — CHANGES_REQUESTED** [review]
minor fixes of typos+ ordering consistency advice.

File: md/release-notes.md (hunk 3 above)
Comment (educhastenier) [reply md/release-notes.md:141]: Done

**Pull request merged**